pkg/asset/installconfig/gcp/client.go

```diff
@@ -60,28 +61,31 @@ func (c *Client) GetNetwork(ctx context.Context, network, project string) (*comp
 }
 
 // GetPublicDomains returns all of the domains from among the project's public DNS zones.
-func (c *Client) GetPublicDomains(ctx context.Context, project string) ([]string, error) {
+func (c *Client) GetPublicDomains(ctx context.Context, project string) (map[string][]string, error) {
```

Comment:
I think this needs an update of the comment to provide details of the new map that is returned.

Reply:
I have updated the comment with a better description of what the map holds.